test/CMakeLists.txt

```diff
@@ -156,7 +159,8 @@ add_spvtools_unittest(
 add_spvtools_unittest(
   TARGET c_interface_shared
   SRCS c_interface_test.cpp
-  LIBS ${SPIRV_TOOLS}-shared)
+  LIBS ${SPIRV_TOOLS}-shared
+  ENVIRONMENT PATH=$<TARGET_FILE_DIR:${SPIRV_TOOLS}-shared>)
```

Comment:
This is needed Windows so it can find the DLL.
I wonder why Linux (and OSX) don't need a modification for LD_LIBRARY_PATH?

Reply:
I think the answer is `RUNPATH`:

```
$ objdump -x test_c_interface_shared | grep PATH
  RUNPATH              /home/joseph/src/vulkan/SPIRV-Tools/build/source
```